pkg/mobility/rrc.go

```diff
@@ -7,7 +7,7 @@ package mobility
 import (
 	"context"
 	"github.com/onosproject/onos-api/go/onos/ransim/types"
-	e2sm_mho "github.com/onosproject/onos-e2-sm/servicemodels/e2sm_mho_go/v1/e2sm-mho-go"
+	e2sm_mho "github.com/onosproject/onos-e2-sm/servicemodels/e2sm_mho_go/v2/e2sm-mho-go"
```

Comment:
It is better to follow golnag Convention for renaming the modules. 
Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.